stashcp/__init__.py

```diff
@@ -671,7 +760,9 @@ def main():
     parser.add_option('-r', dest='recursive', action='store_true', help='recursively copy')
     parser.add_option('--closest', action='store_true', help="Return the closest cache and exit")
     parser.add_option('-c', '--cache', dest='cache', help="Cache to use")
-    parser.add_option('-j', '--caches-json', dest='caches_json', help="The JSON file containing the list of caches",
+    parser.add_option('-j', '--caches-json', dest='caches_json', help="A JSON file containing the list of caches",
+                      default=None)
+    parser.add_option('-n', '--cache-list-name', dest='cache_list_name', help="Name of pre-configured cache list to use",
```

Comment:
This "pre-configured cache-list" is it from the wlcg-wpad, why is the default to None? and can you add an example of what other possible values are?

Reply:
It is from the wlcg-wpad.  None menas to take the first list.  I hesitate to code an example into the help because we'll probably want to change what's in there.  Currently the two examples are "xroot" (the default) and "xroots".  I'm more inclined to add an option to print what the alternatives are.

Reply:
I added a --list-names option to print the names of the preconfigured lists